### ISSUE TITLE
MegaMenu | Keyboard navigation is broken

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#15110]
Keyboard navigation doesn't work as expected.

To replicate:

1-) Focus on the menu
2-) Open a root item
3-) Try to navigate by using arrow keys.
